### DISCUSSION TITLE
DNS Ltsp client debian 11

### DISCUSSION THREAD

**Discussion by svetoi1985** [ltsp/ltsp#813] [Q&A]
Hello, I can't figure out how to register in ltsp.conf the autorun script from the client to record the name in dns.

**Comment by alkisg** [comment]
Not sure what that means. Go to your DNS server and create entries for all the LTSP clients, then they'll be in DNS even if they're not powered on.

**Comment by svetoi1985** [comment]
I have a script for writing to dns, if I use it when the client is loaded, it writes the client's name to dns, which needs to be done in Ltsp.conf so that it runs and is applied before entering the user's password. I have the script in a separate file.

**Comment by alkisg** [comment]
It feels like you're following the wrong solution to solve your DNS needs, but anyway I think you're looking for this:

```ini
[clients]
POST_SERVICE_DNS="/path/to/your/dns/script"
```

Remember to run `sudo ltsp initrd` after modifying ltsp.conf.

**Comment by svetoi1985** [comment]
Thank you so much for helping out. Everything works.
